Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md

```diff
@@ -1,10 +1,170 @@
+    * Added two separator `GameObject`s for scene readability: runtime `NetworkObject`s and `NetworkObject`s already placed in the scene.
+    * Added a custom editor for GameEvents to fire in the editor (greatly enhances testing).
+    * The `LifeState` NetworkVariable was moved from `NetworkCharacterState` into its own component, `NetworkLifeState`.
+    * Cleaned up and removed old spawn prefab collections and spawner scripts (`NetSpawnPoint`).
+
+* Updated the user interface including the following:
+
+  * When joining a game, a "Connecting..." UI loads. When disconnecting from a game, you are returned to the MainMenuScene with a "Connection to Host lost" message. If the game fails to connect, a general message "Connection to Host failed" loads. 
+  * Added an option to leave the lobby after joining it. You no longer have to restart the game to leave the lobby.
+  * Added option to cancel the game after clicking **Ready**. 
+  * Added a gear icon for accessing and modifying app settings. 
+  * The UI now tracks and displays player has arrived, number of connected players in the lobby, and player status. 
+
+* Added F/X and animation assets for the game including:
+
+  * Audio files for boss sound effects
+  * Ramp-up animation for hero movement and actions
+  * Visual effects and particles for Tank charge skill 
+  * Art assets to wave spawner, including animations for ReceiveDamage, Broken (died), and Revive 
+  * Game music 
+
+* Added audiomixer to Boss Room for separate audio channels and a master mixer. 
+
+* Updated and added various hero abilities:
+
+  * Added a cooldown to Archer's PowerShot. 
+  * Added the Rogue's Dagger and Dash skills. 
+  * Added the Rogue's Sneak skill using local stealth, applying a graphical effect to the stealthy character while still sending all network events normally.
+  * Properly display Heal abilities when targeting a fallen ally character. 
+  * Character attack actions properly support Hold to charge options. 
+
+* To show how UI elements and game objects can be networked, added networked functionality using `INetworkSerializable` in the `CharSelect` screen to network player's selected character on the Character Selection screen. 
+* Added a filter to remove any invisible characters introduced by Zoom when copy-pasting room names, IP addresses, or ports. 
+* Boss Room now uses the [UnityToonShader](https://github.com/IronWarrior/UnityToonShader) for rendering the 3D surfaces to emulate 2D, flat surfaces.
+* Added disconnection error message to load when a player or host disconnects due to limited or no network connectivity. Client logic was also updated to detect Host disconnection scenarios, such as losting connectivity.
```

Comment:
```suggestion
* Added disconnection error message to load when a player or host disconnects. Client logic was also updated to detect Host disconnection scenarios, such as losting connectivity.
```

Reply:
we don't support bandwidth issue detection right now

Reply:
Sounds good! Thanks!